data-model/fdo-profiles/0.1.0/virtualcollections/schema/virtualcollections.json

```diff
@@ -0,0 +1,31 @@
+
+{
+  "$id": "https://json-schema.org/draft/2020-12/schemas/fdo-profiles/0.1.0/mas.json",
```

Comment:
change id to "https://json-schema.org/draft/2020-12/schemas/fdo-profiles/0.1.0/virtualcollections.json"

Reply:
id should point to the location on schemas.dissco.tech, I'm aware there's a few places that need fixing.